common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -899,7 +899,9 @@ def test_change_mode_invalid_user(self):
         self.assert_enrollment_status(username='fake-user', expected_status=status.HTTP_406_NOT_ACCEPTABLE,
                                       as_server=True)
 
-    def test_update_enrollment_with_expired_mode_throws_error(self):
+    @ddt.data(True, False)
+    @ddt.unpack
+    def test_update_enrollment_with_expired_mode(self, server_call):
```

Comment:
For clarity, you may want to name this variable something like `using_api_key`. The API key doesn't necessarily indicate a server-server call. Looks fine otherwise.